shared/constants/teams.js

```diff
@@ -84,6 +85,15 @@ const getTopicFromConvID = (state: TypedState, conversationIDKey: ChatTypes.Conv
 const getRole = (state: TypedState, teamname: Types.Teamname): ?Types.TeamRoleType =>
   state.entities.getIn(['teams', 'teamNameToRole', teamname], null)
 
+const getCanPerform = (state: TypedState, teamname: Types.Teamname): I.Map<string, boolean> => {
+  // make a default map
+  var defaultOps = I.Map()
```

Comment:
This is why its confusing to use the 2 kinds of maps. The set() call below doesn't mutate this map at all, it returns a new map